src/parachain/issue.ts

```diff
@@ -149,21 +150,22 @@ export class DefaultIssueAPI implements IssueAPI {
 
     }
 
-    async execute(issueId: H256, txId: H256Le, merkleProof: Bytes, rawTx: Bytes): Promise<void> {
-        if (!this.account) {
-            return Promise.reject(ACCOUNT_NOT_SET_ERROR_MESSAGE);
+    async execute(requestId: string, btcTxId: string, merkleProof?: Bytes, rawTx?: Bytes): Promise<void> {
+        const parsedRequestId = this.api.createType("H256", "0x" + requestId);
+        const parsedBtcTxId = this.api.createType(
+            "H256",
+            "0x" + Buffer.from(btcTxId, "hex").reverse().toString("hex")
```

Comment:
Minor nit: You can shorten this to: `Buffer.from(btcTxId, "hex").reverse()`

See UI for example https://github.com/interlay/polkabtc-ui/blob/dev/src/pages/app/issue/modal/status-view.tsx#L66

Happy to also do that in a future PR.

Reply:
This PR is already fairly big, so I think it's best to do this in a separate PR (coupled with the `transfer`/`BN` return value)